lib/plexy/logger.ex

```diff
@@ -56,7 +67,7 @@ defmodule Plexy.Logger do
   """
   def measure(metric, fun) do
     {time, result} = :timer.tc(fun)
-    debug(%{metric_name(metric, :measure) => time / 1000.0})
+    measure(metric, time / 1000.0)
```

Comment:
Is measure taking a "metric" or the "name" of a measurement?

Milliseconds is built into the documentation for the function, but that isn't guaranteed to be represented in the output measure#. i.e. if the name of the measurement I call this function with is `foo`, should my resulting measure be `measure#foo` or `measure@foo.ms` ? If `ms` isn't included by this function, how does that affect callers and down-stream consumers of the metrics produced?

Reply:
By not adding `ms` to the it allows the callers to set whatever they want `query.sec`, `latency.µs`. Maybe at some point we should discuss standardizing on what type of time we use for all services?